addons/event/models/event_ticket.py

```diff
@@ -133,20 +172,26 @@ def _check_seats_availability(self, minimal_availability=0):
                                   + '\n%s\n' % '\n'.join(sold_out_tickets))
 
     @api.depends('seats_max', 'seats_available')
-    @api.depends_context('name_with_seats_availability')
+    @api.depends_context('name_with_seats_availability', 'name_with_slot_date')
     def _compute_display_name(self):
-        """Adds ticket seats availability if requested by context."""
-        if not self.env.context.get('name_with_seats_availability'):
-            return super()._compute_display_name()
+        """Adds ticket seats availability or ticket slot date (only if slot ticket) if requested by context."""
         for ticket in self:
-            if not ticket.seats_max:
-                name = ticket.name
+            if not self.env.context.get('name_with_slot_date') and ticket.parent_ticket_id:
+                ticket_name = ticket.parent_ticket_id.name
+            else:
+                ticket_name = ticket.name
+            if not self.env.context.get('name_with_seats_availability'):
+                ticket.display_name = ticket_name
+                continue
+            # Name with availability
+            if not ticket.seats_max and not (ticket.slot_id and ticket.event_id.seats_max):
```

Comment:
There is something a bit strange with the right part inside the parentheses. Should we not have the ticket.seats_max depend on event_id.seat_max as well, and only check ticket.seats_max ?